sql_test.go

```diff
@@ -193,6 +194,21 @@ func testNullUUIDScanValid(t *testing.T) {
 	}
 }
 
+func testNullUUIDScanUUID(t *testing.T) {
+	s := UUID{}
```

Comment:
Let's not use the zero value here. It could lead to confusion. Please use `codecTestUUID` instead.

Reply:
done